pkg/sloop/server/internal/config/config_test.go

```diff
@@ -0,0 +1,50 @@
+package config
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"testing"
+	"encoding/json"
+	"github.com/ghodss/yaml"
+)
+
+func Test_loadFromJSONFile(t *testing.T) {
+	var expectedconfig SloopConfig
+	configfilename:="testconfig.json"
+	configFile, err := ioutil.ReadFile(configfilename)
+	err = json.Unmarshal(configFile, &expectedconfig)
+	assert.Nil(t,err)
+
+	out_config,err := loadFromFile(configfilename)
+
+	assert.Nil(t,err)
+	assert.Equal(t,out_config,&expectedconfig)
+}
+
+func Test_loadFromYAMLFile(t *testing.T) {
+	var expectedconfig SloopConfig
+	configfilename:="testconfig.yaml"
+	configFile, err := ioutil.ReadFile(configfilename)
+	err = yaml.Unmarshal(configFile, &expectedconfig)
+	assert.Nil(t,err)
+
+	out_config,err := loadFromFile(configfilename)
+
+	assert.Nil(t,err)
+	assert.Equal(t,out_config,&expectedconfig)
+}
+
+func Test_loadFromTxtFile(t *testing.T) {
+	configfilename:="testconfig.txt"
+	_,err := loadFromFile(configfilename)
+
+	assert.Nil(t,err)
```

Comment:
formatting -> assert.Nil(t, err)
please run `make` it should run `go fmt` and format it for you.

**btw should this be assert.Panics**? Seems like loadFromFile function panics on error from these checks:

if strings.Contains(filename, ".yaml") {
		err = yaml.Unmarshal(configFile, &config)
	} else if strings.Contains(filename, ".json"){
		err = json.Unmarshal(configFile, &config)

Reply:
rather than creating this file, should we just put the json/yaml file content as a constant and calling it in the unit test, such as 
```
test= `
{
	"ConfigFile": "",
	"leftBarLinks": null,
	"resourceLinks": null,
	"disableKubeWatch": false,
	"kubeWatchResyncInterval": 1800000000000,
	"webfilesPath": "./pkg/sloop/webfiles",
	"bindAddress": "",
	"port": 8080,
	"storeRoot": "./data",
	"maxLookBack": 12
...
}
`
```
This way we will have more testing case and dont need to create any of the files, also i would recommend we put the unit test in a separate PR or work item, so we get this bug rolled in to unblock customers, also I think our ic needs sometime to learn how to use mock probably, that might take her some time to learn and write code